### ISSUE TITLE
Container: v1beta1 Cluster class is missing the releaseChannel field

### ISSUE THREAD

**Issue by edigaryev** [googleapis/python-container#13]
This code:

```python
from google.cloud.container_v1beta1 import ClusterManagerClient


PROJECT_ID = "..."
ZONE = "..."

client = ClusterManagerClient()

cluster = {
    "name": "release-channel-test",
    # https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.Channel
    "releaseChannel": 2,
}

client.create_cluster(project_id=PROJECT_ID, zone=ZONE, cluster=cluster)
```

Fails with the following exception:

```
ValueError: Protocol message Cluster has no "releaseChannel" field.
```

`release_channel` doesn't work either:

```
ValueError: Protocol message Cluster has no "release_channel" field.
```

This field seems to be present in the v1beta1 API (https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster), but not in the library.

Library versions:

```
$ pip list | grep google
google-api-core          1.16.0     
google-auth              1.11.2     
google-cloud-container   0.4.0      
googleapis-common-protos 1.51.0     
grpc-google-iam-v1       0.12.3     
```

**Comment by busunkim96** [comment]
Hi @edigaryev, it looks like the library needs an update. The changes will go out in the next release.

**Comment by edigaryev** [comment]
Thanks @busunkim96!

>The changes will go out in the next release.

Any idea how long it might take?

**Comment by busunkim96** [comment]
Hi @edigaryev,

I'll cut a release before the end of the week.

**Comment by busunkim96** [comment]
It looks like I was mistaken. Cluster does not have a `release_channel` field in the v1beta1 proto files. The client libraries are generated from these proto files https://github.com/googleapis/googleapis/blob/fea22b1d9f27f86ef355c1d0dba00e0791a08a19/google/container/v1beta1/cluster_service.proto#L986-L1278 

I'm reaching out to the team internally to see when they're planning an update

**Comment by busunkim96** [comment]
Googlers, see internal issue 154160334 for status of proto update.

**Comment by edigaryev** [comment]
@busunkim96 any updates?

**Comment by busunkim96** [comment]
@edigaryev I don't see any activity on the issue. I just pinged the backend engineer to ask for a rough timeline.

If you have a support contract with Google, please reach out that way as well to help raise the priority.

**Comment by narner90** [comment]
The Cluster class is also missing the `AutoscalingProfile` field within the `ClusterAutoscaling` message, among a few others such as `ShieldedNodes`. Any news when these will be updated?

**Comment by busunkim96** [comment]
Closed by #55. Changes will be available in `>=2.3.0`
